File: Problemas/1915A - Odd One Out/1915A_Odd_One_Out.cpp
// 1915A - Odd One Out
// https://codeforces.com/problemset/problem/XXX/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) pruebas = Número de casos de prueba
    int pruebas;
    cin >> pruebas;

    // Itera sobre cada caso de prueba
    for(int indice_prueba = 0; indice_prueba < pruebas; indice_prueba++) {

        // Input:
        // 2) a,b,c = Dígitos
        int a,b,c;
        cin >> a >> b >> c;

        // Determina cuál es el par de número igual
        // Imprime el número disinto (el que no se comparó)
        if(a == b) {
            cout << c << endl;
        } else if(b == c) {
            cout << a << endl;
        } else if(a == c) {
            cout << b << endl;
        }
    }

    return 0;
}

File: Problemas/1915A - Odd One Out/1915A_Odd_One_Out.py
# 1915A - Odd One Out 
# https://codeforces.com/problemset/problem/1915/A

# Input:
# 1) pruebas = Número de casos de prueba
pruebas = int(input())

# Itera sobre cada caso de prueba
for indice_prueba in range(0,pruebas):

    # Input:
    # 2) a,b,c = Dígitos
    a, b, c = map(int, input().split())

    # Meter los dígitos en una lista
    lista_digitos = [a,b,c]
    
    # Diccionario de frecuencias de dígitos
    instancias_digitos = {}

    # Cuenta cuántas veces aparece cada dígito
    for digito in lista_digitos:
        if digito in instancias_digitos:
            instancias_digitos[digito] += 1
        else:
            instancias_digitos[digito] = 1

    # Extrae las llaves y valores del diccionario de frecuencias
    instancias_digitos_llaves = list(instancias_digitos.keys())
    instancias_digitos_frecuencia = list(instancias_digitos.values())

    # Extrae el número distinto (cuya frecuencia es igual a 1)
    numero_distinto = instancias_digitos_frecuencia.index(1)

    # Imprime el número distinto
    print(instancias_digitos_llaves[numero_distinto])
